src/controllers/clientController.js

```diff
@@ -16,15 +16,15 @@ exports.createClient = async (req, res) => {
       throw new Error('Client already exists');
     }
 
-    const createdClient = await Client.create({
+    await Client.create({
       userId: decodedToken.userId,
       address,
       contactNumber,
     });
 
-    return res.status(201).json(createdClient);
+    res.status(200).redirect('/api/clients/chefs');
```

Comment:
I thought that it might make more sense, so after the user fills in his client info then he will be redirected to his dashboard which is a page that brings him the available chefs